machine/wnc_x86_64/wnc_onie_build.sh

```diff
@@ -0,0 +1,28 @@
+#! /bin/sh
```

Comment:
Private build scripts are not accepted as it breaks the general documentation about how to build.

This type of custom build script is better kept in a separate repository on your end.